Automattic/wp-calypso
# Update wpcom block editor welcome modal contents

Closes https://github.com/Automattic/wp-calypso/issues/40739

Restyle and update contents of the WPCOM block editor nux flow.

- Add and enqueue style (to do item to dynamic import from JS).
- Add public path so image assets can be found.
- Replace JS images with dynamic webpack image paths.
- Restyle and update contents of the NUX modal. See #40739

To do: 
- Navigation buttons need improvement on mobile. (postpone -- less critical as long as the last step (private NUX) isn't there)
- [x] Change texts in Gutenboarding flow - https://github.com/Automattic/wp-calypso/pull/40828#discussion_r404398989
- Dynamically load CSS on render. (can be postponed)
- Conditionally show private NUX only for private sites. (postpone https://github.com/Automattic/wp-calypso/pull/40828#discussion_r404770910)
- [x] ⚠️ Remove testing commit! ⚠️ f44e52c

## Testing instructions

- Sandbox a simple site (`example.wordpress.com`) and apply D41466-code.
- Go to `https://wordpress.com/block-editor/post/<yourSite>?is-gutenboarding`
- Manually activate the welcome modal via this menu:

<img width="379" alt="Screenshot 2020-04-03 at 09 55 28" src="https://user-images.githubusercontent.com/87168/78333354-5b8f1480-7592-11ea-9c79-7977ddeb9fac.png">

- Verify that the steps look as below.
- Try the same URL but without `?is-gutenboarding`.

## Screens
### Pretty small (<660px)
![small](https://user-images.githubusercontent.com/841763/78699261-e9a52b00-7903-11ea-8862-436b047d870e.gif)

### Medium
![med](https://user-images.githubusercontent.com/841763/78699267-eb6eee80-7903-11ea-8487-061a8513047d.gif)

### Big!
![large](https://user-images.githubusercontent.com/841763/78699272-edd14880-7903-11ea-97bb-2e74b6acceb9.gif)
